c2/c2-protocol/c2-protocol-api/src/main/java/org/apache/nifi/c2/protocol/api/FlowInfo.java

```diff
@@ -31,6 +31,7 @@ public class FlowInfo implements Serializable {
     private Map<String, ComponentStatus> components;
     private Map<String, FlowQueueStatus> queues;
     private List<ProcessorBulletin> processorBulletins;
+    private List<ProcessorStatus> processors;
```

Comment:
I think it would be more descriptive to call this member processorStatuses and also rename it's accessors

Reply:
I followed the patter what was used for 'queues' above but for sure I can change the property name.

Reply:
Done